### PR TITLE
Add Muse DAO CW20 from JUNO

### PR DESCRIPTION
**Pull request by reza2870**
Please add MUSE token to osmosis!
Discord: https://discord.gg/7GRK8mcp

### CODE REVIEW

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -3244,6 +3244,36 @@
...
       }
-    }
+    },
+	{
+      "description": "MUSE Governance Token",
+      "type_asset": "cw20",
+      "address": "juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3",
+      "denom_units": [
+        {
+          "denom": "IBC/6B982170CE024689E8DD0E7555B129B488005130D4EDA426733D552D10B36D8F",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:3254]: please make the letter: 'IBC' lower case. -> "ibc/6B9...

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 2]
```diff
@@ -3244,6 +3244,36 @@
...
+      "name": "MUSE",
+      "display": "muse",
+      "symbol": "MUSE",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "cw20:juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/MUSE.png",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:3273]: please point these img urls instead to the chain registry. E.g.,
"https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/muse.svg"

Comment (reza2870) [reply osmosis-1/osmosis-frontier.assetlist.json:3273]: Need to PR to upload image at this path?

Comment (reza2870) [reply osmosis-1/osmosis-frontier.assetlist.json]: updated

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 1 above)
Comment (reza2870) [reply osmosis-1/osmosis-frontier.assetlist.json]: updated

**Comment by reza2870** [comment]
ibc and image url updated

File: osmosis-1/osmosis-frontier.assetlist.json (hunk 2 above)
Comment (JeremyParish69) [reply osmosis-1/osmosis-frontier.assetlist.json]: would be best to have these imgs at the Chain Registry, yes. I only see muse.png there...

Comment (reza2870) [reply osmosis-1/osmosis-frontier.assetlist.json]: I must create new PR for svg?
can we link for svg this site?

Comment (reza2870) [reply osmosis-1/osmosis-frontier.assetlist.json]: I deleted svg.

**Comment by reza2870** [comment]
updated

**Comment by reza2870** [comment]
Please review PR

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 3]
```diff
@@ -3244,6 +3244,35 @@
...
+      "symbol": "MUSE",
+      "ibc": {
+        "source_channel": "channel-47",
+        "dst_channel": "channel-169",
+        "source_denom": "cw20:juno1p8x807f6h222ur0vssqy3qk6mcpa40gw2pchquz5atl935t7kvyq894ne3"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/muse.png"
+      }
+    },
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-frontier.assetlist.json:3275]: I think the comma here is causing the schema validation to fail

Comment (reza2870) [reply osmosis-1/osmosis-frontier.assetlist.json:3275]: ok. I remove now

**Pull request merged**